src/Http/Routing/src/Builder/EndpointRouteBuilderExtensions.cs

```diff
@@ -88,6 +88,22 @@ public static class EndpointRouteBuilderExtensions
             return MapMethods(endpoints, pattern, DeleteVerb, requestDelegate);
         }
 
+        /// <summary>
+        /// Adds a <see cref="RouteEndpoint"/> to the <see cref="IEndpointRouteBuilder"/> that matches HTTP PATCH requests
+        /// for the specified pattern.
+        /// </summary>
+        /// <param name="endpoints"></param>
```

Comment:
Nit: these params need to be filled in.

Reply:
Added comments.
Thanks @captainsafia 🐱‍🏍